docs/howtos/syn_hkdbs.rst

```diff
@@ -0,0 +1,145 @@
+.. _sec-syn-hk-dbs:
+
+============================================
+        Using synthetic diagnostic
+============================================
+
+The following discusses how one uses the synthetic diagnostic for ``pyrokinetics``. The steps in the synthetic diagnostic are the following:
+1. Inputs are diagnostic specific (diagnostic, filter, k, location, resolution, local rhos). See example_syn_hk_dbs.py
+2. Load equilibrium, kinetics files. Find scattering location theta. See __init__ in class SyntheticHighkDBS
+3. Map (kn, kb) to (kx, ky) for all k's / channels specified in 1. See function mapk
+4. Load GK output data (fluctuation moment file). See function get_syn_fspec 
+5. For each input condition (eg. for each k in highk/DBS), filter simulation data. See class Filter, functions apply_filter, get_syn_fspec
+6. Generate synthetic spectra and make plots. See functions get_syn_fspec, plot_syn
+
+
+1. We first point is to provide input data from the specific diagnostic (high-k, DBS for now). See example_syn_hk_dbs.py as reference. 
```

Comment:
```suggestion
## 1. 
We first point is to provide input data from the specific diagnostic (high-k, DBS for now). See example_syn_hk_dbs.py as reference. 
```

Reply:
I made a typo, 'The first point is ...' not 'We first point is ...'.